src/Microsoft.Unity.Analyzers.Tests/Microsoft.Unity.Analyzers.Tests.csproj

```diff
@@ -20,4 +20,13 @@
     <ProjectReference Include="..\Microsoft.Unity.Analyzers\Microsoft.Unity.Analyzers.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Reference Include="Microsoft.CodeAnalysis.CodeStyle">
```

Comment:
Have you tried, to include it as a <None> file with a copy to output set to always instead of as a binary reference?

Reply:
I tried, it works indeed.